### ISSUE TITLE
Use standard terminology for inherited state

### ISSUE THREAD

**Issue by jeromekelleher** [jeromekelleher/sc2ts-paper#173]
We're using things like "initial state" in the captions for the supplemental subgraph figs. Standardise to stated conventions and ref back to section defined

**Comment by hyanwong** [comment]
Good point. Do we want to add a subsubsection before the new paragraph starting "Tskit supports arbitrarily complex patterns of mutation at a particular site", so that we can refer back to that specific para easily?

**Comment by jeromekelleher** [comment]
I don't think it's worth it, can just refer back to the ARGs section. It's only a few paras, won't be that hard to find
